src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -75,6 +75,8 @@ def load_arguments(self, _):
             c.argument('disable_key_based_metadata_write_access', arg_type=get_three_state_flag(), help="Disable write operations on metadata resources (databases, containers, throughput) via account keys")
             c.argument('enable_public_network', options_list=['--enable-public-network', '-e'], arg_type=get_three_state_flag(), help="Enable or disable public network access to server.")
             c.argument('enable_analytical_storage', arg_type=get_three_state_flag(), help="Flag to enable log storage on the account.", is_preview=True)
+            c.argument('network_acl_bypass', options_list=['--network-acl-bypass'], help="Flag to enable or disable Network Acl Bypass.")
```

Comment:
which value would be allowed to enable or disable acl bypass? If they are enum values, could you use `arg_type=get_enum_type(xxx)` here to show customer allowed values? If it is boolean, consider using `arg_type=get_three_state_flag()`.

Reply:
It is an enum, I added get_enum_type(NetworkAclBypass)